### ISSUE TITLE
[low] Color Contrast: Set password: Grey text over light grey background fails minimum contrast requirements

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5512]
## Action Performed:
1. Using Chrome, open https://staging.new.expensify.com/
2. Navigate to the 'Enter your phone or email' field and enter a valid email address, then click on 'Continue' button.
3. Open the received email and click on 'here' link.
4. Once the 'Set password' page opens, measure the color contrast ratio of the grey 'Your password must have at least 8 characters, 1 capital letter, 1 lowercase letter, 1 number' text against light grey background.

## Expected Result:
The contrast ratio of the text against the background should be at least 4.50:1 for small text and 3.00:1 for large text (14pt bold or 18 pt+).

## Actual Result:
The grey 'Your password must have at least 8 characters, 1 capital letter, 1 lowercase letter, 1 number' text (#7D8B8F) against light grey background (#FAFAFA) has a color contrast ratio of 3.37:1. This text is small (14px=10.5pt).

HTML:
`<div dir="auto" class="css-901oao" style="color: rgb(125, 139, 143); font-family: GTAmericaExp-Regular; font-size: 13px; line-height: 18px; margin-top: 4px;">Your password must have at least 8 characters,
1 capital letter, 1 lowercase letter, 1 number.</div>`

CSS:
`.css-901oao {
border: 0px solid black;
box-sizing: border-box;
color: rgb(0, 0, 0);
display: inline;
font: 14px -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
margin: 0px;
padding: 0px;
white-space: pre-wrap;
overflow-wrap: break-word;
}`

## Workaround:
No workaround

## Area issue was found in:
Set password page

## Failed WCAG checkpoints
1.4.3

## User impact:
The low-vision and color-blind users might not be able to see the text.

## Suggested resolution:
Use a darker shade of grey for text, such as `#737373`, which will give a color contrast ratio of 4.54:1.

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5182626_color_contrast_-_grey_text](https://user-images.githubusercontent.com/88733897/134731933-db713c14-1895-4fef-8191-a8d2e997ea93.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
